pkg/manifests/manifests_test.go

```diff
@@ -1433,30 +1433,43 @@ func TestPrometheusQueryLogFileConfig(t *testing.T) {
 			errExpected:      true,
 			volumeExpected:   false,
 		},
-	}
-	for _, tt := range testCases {
+		{
+			name:             "invalid path, relative path",
+			queryLogFilePath: "./dev/query.log",
+			expected:         "",
+			errExpected:      true,
+			volumeExpected:   false,
+		},
+		{
+			name:             "filename only",
+			queryLogFilePath: "query.log",
+			expected:         "query.log",
+			errExpected:      false,
```

Comment:
Isn't `query.log` a relative path?

Reply:
if `queryLogFile` doesn't contain a directory (absolute or relative), the Prometheus operator will add and automatically mount an emptyDir volume (see https://github.com/prometheus-operator/prometheus-operator/pull/4566).